esi_leap/api/controllers/v1/offer.py

```diff
@@ -58,13 +60,19 @@ def __init__(self, **kwargs):
 class OffersController(rest.RestController):
 
     @wsme_pecan.wsexpose(Offer, wtypes.text)
-    def get_one(self, offer_uuid):
+    def get_one(self, offer_id):
         request = pecan.request.context
         cdict = request.to_policy_values()
         policy.authorize('esi_leap:offer:get', cdict, cdict)
 
-        o = offer.Offer.get(request, offer_uuid)
-        o = OffersController._add_offer_availabilities(o)
+        o_id = offer.Offer.get(offer_id)
+
+        if len(o_id) > 1:
+            raise exception.OfferDuplicateName(name=offer_id)
+        elif len(o_id) == 0:
+            raise exception.OfferNotFound(offer_uuid=offer_id)
```

Comment:
I noticed that you check the `offer_admin` policy in `delete`, but not here - any reason why not? If you end up checking it here as well, then that logic can probably be encapsulated in a function, similarly to the contract code.

Reply:
The reason I do not check any policy here is because we had decided that anybody can view any offer.

Reply:
Obviously anybody should not be able to delete any offer randomly so I use policy in delete()

Reply:
Ah, you're 100% correct!